utils/service_discovery/sd_docker_backend.py

```diff
@@ -111,6 +111,9 @@ def _make_fetch_state(self):
     def update_checks(self, changed_containers):
         state = self._make_fetch_state()
 
+        if Platform.is_k8s():
+            self.kubeutil.check_services_cache_freshness()
```

Comment:
added it, and added some code to check__freshness to skip duplicate checks if it were to happen